### PR TITLE
lightning: disk quota GA

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

The disk quota feature goes GA.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/12090, https://github.com/pingcap/docs-cn/pull/12162
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by niubell — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 1]
```diff
@@ -224,10 +224,6 @@ The [`num-threads`](/tikv-configuration-file.md#num-threads) configuration of Ti
 
 ## Configure disk quota <span class="version-mark">New in v6.2.0</span>
 
-> **Warning:**
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:227]: Same suggestion as the DM continuous validation feature.

**Review by shichun-0415 — COMMENTED** [review]
Rest LGTM

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md (hunk 1 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-physical-import-mode-usage.md:227]: done

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by dsdashun — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -154,6 +154,14 @@ addr = "172.16.31.10:8287"
 # node in the physical import mode. 0 by default, which means no limit.
 # store-write-bwlimit = "128MiB"
 
+# Specifies the disk quota for the local temporary files when using the Physical Import Mode.
+# When the disk quota is insufficient, TiDB Lightning pauses the process of reading the source data and writing the temporary files,
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-configuration.md:158]:
```suggestion
# Specifies the disk quota for local temporary files when physical import mode is used.
# When the disk quota is insufficient, TiDB Lightning stops reading source data and writing temporary files,
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -154,6 +154,14 @@ addr = "172.16.31.10:8287"
...
+# and prioritizes writing the already sorted key-value pairs to TiKV.
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-configuration.md:159]:
```suggestion
# but prioritizes writing the already sorted key-value pairs to TiKV.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -331,6 +339,9 @@ compact = false
 switch-mode = "5m"
 # Duration between which an import progress is printed to the log.
 log-progress = "5m"
+# When you use the Physical Import Mode, the time interval for checking the local disk quota.
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-configuration.md:342]:
```suggestion
# The time interval for checking the local disk quota when you use the physical import mode.
```

**Pull request merged**